amads/algorithms/__init__.py

```diff
@@ -19,7 +19,7 @@
 from ..polyphony.skyline import skyline
```

Comment:
As noted by email, maybe you could take this opportunity to remove `music21` from the list of dependencies in pyproject.toml?

Reply:
Great, will do in a separate PR in case of unforeseen chaos ;)